core-bundle/tests/Image/ImageSizesTest.php

```diff
@@ -164,6 +172,28 @@ public function testReturnsTheRegularUserOptions(): void
         $this->assertSame([], $options);
     }
 
+    public function testServiceIsResetable(): void
+    {
+        $this->assertInstanceOf(ResetInterface::class, $this->imageSizes);
+
+        $this->framework
+            ->expects($this->exactly(2))
+            ->method('initialize')
+        ;
+
+        $this->connection
+            ->expects($this->exactly(2))
+            ->method('fetchAll')
+            ->willReturn([])
+        ;
+
+        $this->imageSizes->getAllOptions();
+        $this->imageSizes->getAllOptions();
```

Comment:
Is this double method call correct?

Reply:
Yes. We‘re calling the method 3 times but expect that the DB connection is only called twice (once again after reset).